### PR TITLE
Revert to old spacewind

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The new spacewind is a constant source of complaining even after having months to get used to it. Unless I have seriously misjudged player opinion the new fast wind is almost universally disliked.

### CODE REVIEW

**Comment by xxalpha** [comment]
:-1:

I like people being pushed down hallways by robust atmos flooding.

**Comment by RemieRichards** [comment]
The default github avatar doesn't suit you.
yours looks like an angry face and that's weird.

**Comment by Iamgoofball** [comment]
@KorPhaeron get in #coderbus nerd

**Comment by Anonus** [comment]
KOR RELIVES!
I'd like this if it had a lower move cooldown so it's somewhere between current and old, because sucking people into scrubbers is fun.

**Comment by WJohn** [comment]
I think it'd be better if it didn't drag PEOPLE as hard as it currently does, but I personally like our fastmos mechanics. If this does that, then :+1:

**Comment by MrStonedOne** [comment]
Ideally, this could be something where if the atom is moving (holding down the move button) we could just make it not effect them unless its really high pressure, maybe even make it scale so it slows them down a little bit.

This would require tracking spacewind on tiles and if a movement key is held down on mobs.

This way it can be FAST unless you attempt to fight it.

**Comment by Fox-McCloud** [comment]
You could make this based on the SSair's cycle, as well--which would make it better than what it currently is (it'll still be faster than the implementation in this PR) and prevent some weird double-movements under some conditions.

It also means that how much you get pushed around will scale with how faster LINDA is currently running.

**Comment by Aranclanos** [comment]
I wish you would stand to your word when quitting, its the third time I think. Just go away already and stop pretending to care.
anyways git gud

**Comment by WJohn** [comment]
Can we not close this PR because of completely unrelated things?

**Comment by Cheridan** [comment]
Yeah, just raise the pressure resistance of humans if people don't like it, having a cooldown on movement looks so janky and bad.

**Comment by Lumipharon** [comment]
:+1

**Comment by Fox-McCloud** [comment]
@Cheridan 

If you base it off of the current cycle, it won't look as bad, as it will adjust to how fast LINDA is running---that's how this was handled in the very beginning.

**Comment by Miauw62** [comment]
what happened to "we want more lethality"

**Comment by Lumipharon** [comment]
Because "lethality" is not the same as "oh god I can't walk down a fucking corridor/entire department because someone opened a canister of AIR"

**Comment by Cookiesndip** [comment]
Yes yes yes yes! Fucking finally. Merge it now.

Cheridan - it's not just humans, it's also objects gravitating toward a 1x1 area that has been patched from all the way down the hall. I remember this happening with all the chef's food. EVERYTHING jumped off his counter and ran all the way to right before the escape airlocks to a patched RCD-punched hole.

It's just frustrating. It doesn't add anything positive to this game.

**Comment by Aranclanos** [comment]
the code is horrible don't even think on merging this

**Comment by Iamgoofball** [comment]
What's bad about it, memeclanos?

**Comment by xxalpha** [comment]
It's the wrong way to do it.

File: code/LINDA/LINDA_turf_tile.dm [hunk 1]
```diff
@@ -288,10 +288,14 @@
 
 
 /atom/movable/var/pressure_resistance = 5
+/atom/movable/var/last_forced_movement = 0
```
Comment (Iamgoofball) [inline code/LINDA/LINDA_turf_tile.dm]: huh, didn't know you could define vars like this

The More You Know(tm)

Comment (duncathan) [inline code/LINDA/LINDA_turf_tile.dm]: Defining variables is basically the only time we use relative pathing
And even then we could get far more relative than we are now

Comment (Iamgoofball) [inline code/LINDA/LINDA_turf_tile.dm]: nah, relative pathing vars would be
> obj
>     var
>          pornhub_sponsored

Comment (duncathan) [inline code/LINDA/LINDA_turf_tile.dm]:
```
obj/
    var/x
```
That's technically still relative pathing. Absolute would be `obj/var/x`.

Comment (xxalpha) [inline code/LINDA/LINDA_turf_tile.dm]: How did you not fucking know this goof?

**Comment by Razharas** [comment]
New system in objectively better, if you want to reduce how much humans are pushed by air do it in the new system by increasing their resistance

**Comment by Lumipharon** [comment]
raz have you seen an overpressured corridor with new space wind? Everything, humans and loose items go tumbling down the corridor, around the corner and fuck off to where ever the fuck.

One time the AI had to re-pressurise toxins or some shit, and accidentally the pressure went up to about 200 or something.
The entire science department was an unplayable mess for the rest of the game - even with the AI trying to fix it, you couldnt move anywhere, or even stand still because of the pressure fuckery. I had to line up 3 lockers in R&D so I could actually use the console.

**Comment by xxalpha** [comment]
@Lumipharon we're just saying that the code here is not the best solution, you can already tweak the system by changing some values.

File: code/LINDA/LINDA_turf_tile.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/LINDA/LINDA_turf_tile.dm]: my life is a lie

**Comment by KorPhaeron** [comment]
Tweaking the values would just effect if it kicks in though, not stop the issue at higher pressures when your mob is being thrown wildly back and forth by two different pressure sources.

**Comment by Anonus** [comment]
SPACE WINS, ONCE AND FOR ALL

**Comment by Lumipharon** [comment]
RIP. And now to watch spacewind never get fixed.

**Pull request closed without merging**